core/domain/email_manager.py

```diff
@@ -503,3 +503,48 @@ def send_instant_feedback_message_email(
             recipient_id, feconf.SYSTEM_COMMITTER_ID,
             feconf.EMAIL_INTENT_FEEDBACK_MESSAGE_NOTIFICATION, email_subject,
             email_body, feconf.NOREPLY_EMAIL_ADDRESS)
+
+
+def send_report_email(
+        exploration_title, exploration_id, reporter_id, report_type,
+        exploration_owner_ids):
+    email_subject = 'New report for "%s"' % exploration_title
+
+    email_body_template = (
+        'Hello Moderator,<br>'
+        '%s has submitted a new report on the exploration %s of the type %s, '
```

Comment:
The text of this email may need to be changed, if we are providing the report_text instead of the report_type.